### PR TITLE
Ap Item index

### PR DESCRIPTION
**Pull request by arpariseau**
For 17 and 18. 17 has the proper functionality, but tests are wonky because it's not properly clicking the image. 18 is fully functional, index now shows top 5 most popular and least popular items ordered.

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -12,6 +12,9 @@ class Item <ApplicationRecord
   validates_inclusion_of :active?, :in => [true, false]
   validates_numericality_of :price, greater_than: 0
 
+  def self.sort_popularity(limit, order)
+    Item.select("items.*, sum(quantity) as qty_ordered").joins(:item_orders).group(:id).order("qty_ordered #{order}").limit(limit)
```
Comment (danielpselph) [inline app/models/item.rb:16]: Good use of SQL injection

File: app/models/item.rb [hunk 2]
```diff
@@ -12,6 +12,9 @@ class Item <ApplicationRecord
...
+  end
```
Comment (kiverso) [inline app/models/item.rb:17]: Really like this method.

**Comment by danielpselph** [comment]
Nice work with helper methods and SQL queries. Will continue to work on image testing.

**Review by kiverso — APPROVED** [review]
Looks good

**Pull request merged**